src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/config/ErrorConfiguration.kt

```diff
@@ -123,6 +124,26 @@ class ErrorConfiguration(private val telemetryClient: TelemetryClient) {
     return errorResponse(e.statusCode, "Call to dependency response exception", e.responseBodyAsString, userMessageForWebClientException(e.statusCode))
   }
 
+  @ExceptionHandler(DownstreamApiCallError::class)
+  fun handleDownstreamApiCallError(e: DownstreamApiCallError): ResponseEntity<ErrorResponse> {
+    telemetryClient.trackEvent(
+      "InterventionsDownstreamAPICallError",
```

Comment:
We'd need to know which API failed at this point, without knowing the categories that can exist. Can we name this specific to `community-api` or passing down the `source`-like attribute too, please?

Reply:
Renamed